chain/chain/src/chain.rs

```diff
@@ -3947,11 +3948,15 @@ impl<'a> ChainUpdate<'a> {
         let head = self.chain_store_update.head()?;
         let is_next = block.header().prev_hash() == &head.last_block_hash;
 
-        // A heuristic to prevent block height to jump too fast towards BlockHeight::max and cause
-        // overflow-related problems
-        let block_height = block.header().height();
-        if block_height > head.height + self.epoch_length * 20 {
-            return Err(ErrorKind::InvalidBlockHeight(block_height).into());
+        // Sandbox allows fast-forwarding, so only enable when not within sandbox
+        #[cfg(not(feature = "sandbox"))]
+        {
```

Comment:
This could be written as `if !cfg!(feature = "sandbox") {` which would mena `epoch_length` does not need to be marked as dead code.